iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -56,7 +58,7 @@ public static boolean isHelmEvaluatorExecutableAvailable() {
     return OperatingSystemUtils.getCurrentPlatformIfSupported().isPresent();
   }
 
-  private void initialize() {
+  protected void initialize() {
     try {
```

Comment:
Actually, this code block (lines 62-67) should be moved to `KubernetesSensor.initContext()`. Then the method `initialize()` can be removed which will simplify the overall design.

Reply:
It can be done in next PR